QgisModelBaker/gui/validate.py

```diff
@@ -363,9 +380,10 @@ def _run(self, edited_command=None):
         validator.configuration.skip_geometry_errors = (
             self.skip_geometry_errors_check_box.isChecked()
         )
-        validator.configuration.valid_config = (
-            self.validator_config_file_line_edit.text()
+        validator.configuration.valid_config = self._absolute_path(
+            self.config_file_line_edit.text()
         )
+        print(validator.configuration.valid_config)
```

Comment:
forgotten print? or is it just to check if i look at the changes? 😄

Reply:
yes :-p